lib/park.rb

```diff
@@ -0,0 +1,43 @@
+class Park 
+  attr_reader :name, :admission_price, :vehicles, :passengers
+
+  def initialize(name, admission_price)
+    @name = name
+    @admission_price = admission_price
+    @vehicles = []
+  end
+
+  def add_vehicle(vehicle)
+    @vehicles << vehicle
+  end
+
+  def attendees
+    @vehicles.flat_map {|vehicle| vehicle.passengers}
+  end
+
+  def calculate_revenue
```

Comment:
```suggestion
  def revenue
```
Like `passengers` above, the interaction pattern calls this `revenue` so you should do the same.